### ISSUE TITLE
Update and clean up localized strings for release

### ISSUE THREAD

**Issue by Vogel612** [rubberduck-vba/Rubberduck#1043]
Since the last release there have been several changes to the localized content and Strings. In addition to that the RESX-Files are scheduled for splitting into separate files, depending on the Window they belong to. see #893 for more details on that.

Following languages need checking:

- [x] German
- [x] French
- [ ] Swedish
- [ ] Japanese

This should only happen after all unused RESX-Entries have been removed to minimize the effort for the translators. As such first things first:

- [x] Remove all unused entries from `RubberduckUI.resx`

Purging the unused entries from other locales will be my job. I'll just run the latest TranslationHelper over it. That cleans it up.

**Comment by Hosch250** [comment]
@Vogel612 I just removed several unused entries related to the inspections.  You might want to update the other locales.  I *think* everything else is used, but I did not cover everything with a fine-tooth comb.  It would take hours to comprehensively prove that a value is either used or unused given that we do not always directly access the value, but sometimes get it through the resource manager using dynamic values, such as the names of a selected enum value...
